### PR TITLE
Taking out button group out of notebook#7694

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

File: view/sale_channel_form.xml [hunk 1]
```diff
@@ -51,12 +51,14 @@
                     <field name="magento_taxes" colspan="8"/>
                 </page>   
               </notebook>
-            <group id="magento_buttons" col="3" colspan="4">
+        </page> 
+    </xpath>
+    <xpath expr="/form/notebook" position="after">
+        <group id="action_buttons" col="3" colspan="4" states="{'invisible': Not(Eval('source') == 'magento')}">
                 <button string="Configure Magento Connection" name="configure_magento_connection"/>
```
Comment (tarunbhardwaj) [inline view/sale_channel_form.xml]: fix indentation

File: view/sale_channel_form.xml [hunk 2]
```diff
@@ -50,13 +50,14 @@
...
-            <group id="magento_buttons" col="3" colspan="4">
-                <button string="Configure Magento Connection" name="configure_magento_connection"/>
-                <button string="Import Magento Order States" name="import_magento_order_states"/>
-                <button string="Import Magento Carriers" name="import_magento_carriers"/>
-            </group>
+            </notebook>
         </page> 
     </xpath>
+    <xpath expr="/form/notebook" position="after">
+        <group id="action_buttons" col="3" colspan="4" states="{'invisible': Not(Eval('source') == 'magento')}">
```
Comment (prakashpp) [inline view/sale_channel_form.xml]: why are you creating a new group? Just use the existing one.

File: view/sale_channel_form.xml [hunk 3]
```diff
@@ -50,13 +50,15 @@
...
-              </notebook>
-            <group id="magento_buttons" col="3" colspan="4">
-                <button string="Configure Magento Connection" name="configure_magento_connection"/>
-                <button string="Import Magento Order States" name="import_magento_order_states"/>
-                <button string="Import Magento Carriers" name="import_magento_carriers"/>
-            </group>
+            </notebook>
         </page> 
     </xpath>
+    <xpath expr="/form/group[@id='action_buttons']" position="replace_attributes">
```
Comment (priyankarani) [inline view/sale_channel_form.xml]: no such action_buttons exists previously
check https://github.com/openlabs/trytond-sale-channel/blob/develop/view/sale_channel_form.xml

**Comment by prakashpp** [comment]
This is not good approach.

**Pull request closed without merging**